terminus-ui/logo/src/logo.component.scss

```diff
@@ -0,0 +1,23 @@
+@import './../../scss/helpers/reset';
+@import './../../scss/helpers/color';
+
+
+//
+// @component
+//  Logo
+// @description
+//  Styles for the logo component
+//
+.ts-logo {
+  @include reset;
+
+  // black:
+  &.ts-logo--black svg path {
+    fill: color(pure, dark);
+  }
+
+  // gray :
+  &.ts-logo--gray svg path {
+    fill: #44525e;
```

Comment:
Let's create an SCSS variable just inside the top style (`.ts-logo`) with a comment so we know why a custom color is used.